scripts/TreeSitter_java.patch

```diff
@@ -1,22 +1,13 @@
---- a/generated-sources/jextract/io/github/treesitter/jtreesitter/internal/TreeSitter.java
-+++ b/generated-sources/jextract/io/github/treesitter/jtreesitter/internal/TreeSitter.java
-@@ -55,9 +55,16 @@ public class TreeSitter {
+--- a/target/generated-sources/jextract/io/github/treesitter/jtreesitter/internal/TreeSitter.java	
++++ b/target/generated-sources/jextract/io/github/treesitter/jtreesitter/internal/TreeSitter.java
```

Comment:
The command runs in the build directory, which may or may not be "target".
```suggestion
--- a/generated-sources/jextract/io/github/treesitter/jtreesitter/internal/TreeSitter.java	
+++ b/generated-sources/jextract/io/github/treesitter/jtreesitter/internal/TreeSitter.java
```

Reply:
done